engine/class_modules/priest/sc_priest.cpp

```diff
@@ -803,6 +803,8 @@ void priest_t::create_procs()
   procs.surge_of_light_overflow  = get_proc( "Surge of Light lost to overflow" );
   procs.serendipity              = get_proc( "Serendipity (Non-Tier 17 4pc)" );
   procs.serendipity_overflow     = get_proc( "Serendipity lost to overflow (Non-Tier 17 4pc)" );
+  procs.power_of_the_dark_side   = get_proc("Power of the Dark Side Penance damage buffed");
+  procs.power_of_the_dark_side_overflow = get_proc("Power of the Dark Side lost to overflow");
```

Comment:
can you line up the `=`'s here? so

```
  procs.surge_of_light_overflow         = get_proc( "Surge of Light lost to overflow" );
  procs.serendipity                     = get_proc( "Serendipity (Non-Tier 17 4pc)" );
  procs.serendipity_overflow            = get_proc( "Serendipity lost to overflow (Non-Tier 17 4pc)" );
  procs.power_of_the_dark_side          = get_proc( "Power of the Dark Side Penance damage buffed" );
  procs.power_of_the_dark_side_overflow = get_proc( "Power of the Dark Side lost to overflow" );
``` 
and then add spaces between the parens and quotes